### ISSUE TITLE
SRLinux + vJunos-switch lab-example

### ISSUE THREAD

**Issue by akielaries** [srl-labs/containerlab#1574]
I was curious what the meaning of this line was in the [srlvjunos01](https://github.com/srl-labs/containerlab/blob/main/lab-examples/srlvjunos01/srlvjunos01.clab.yml) lab example:

```
      image: registry.srlinux.dev/pub/vr-vjunosswitch:23.2R1.14
```
and why we wouldn't flag the image as vrnetlab/vr-vjunosswitch:23.2R1.14 instead? I like the idea of being able to pull these VMs as docker images from some public registry but was curious on why you chose this to flag the image.

**Comment by hellt** [comment]
This is an internal name that I accidentally committed. This repo is not available publicly, I'll amend it

**Comment by hellt** [comment]
fixed in [722c201](https://github.com/srl-labs/containerlab/commit/722c201a57c80df38226cb5ac4fd3ebebe703f3c)
